.github/workflows/e2e-tests.yml

```diff
@@ -122,21 +122,31 @@ jobs:
         CONFIG_KEY: ${{ secrets.E2E_CONFIG_KEY }}
         SUITE: ${{ matrix.suite }}
         PROJECT_NAME:  ${{ matrix.project }}
+        DISPATCH_REF_NAME: ${{ github.event.client_payload.ref_name }}
+        DISPATCH_REF_TYPE: ${{ github.event.client_payload.ref_type }}
       run: |
         echo "::group::Decrypt config"
         pnpm run config:decrypt
         echo "::endgroup::"
 
-        if [ "${SUITE}" != atomic ]; then
+        if [[ "${SUITE}" != atomic ]]; then
           echo "::group::Start docker environment"
           pnpm run env:up
           echo "::endgroup::"
 
           echo "::group::Create tunnel"
           pnpm run tunnel:up
           echo "::endgroup::"
-        else
-          echo "TEST_SITE=atomic" >> $GITHUB_ENV
+        fi
+
+        if [[ "${SUITE}" == atomic ]]; then
+          if [[ "${DISPATCH_REF_TYPE}" == tag ]]; then
+            SUITE="atomicRc"
+          fi
+          echo "TEST_SITE=${SUITE}" >> $GITHUB_ENV
+
+          # Wait for the site to pick up latest tag version (DISPATCH_REF_NAME)
+          node ./bin/wait-for-atomic-site.cjs $DISPATCH_REF_TYPE $DISPATCH_REF_NAME
```

Comment:
Here you're in the plugin/tests/e2e path, you need to cd into `tools/e2e-commons` or provide a full path to `wait-for-atomic-site.cjs`. Not sure, but I think the next command works in e2e-commons dir also, so there's no need to cd back.